accounts/forms.py

```diff
@@ -277,6 +277,15 @@ def __init__(self, *args, **kwargs):
         self.user = kwargs.pop('user', None)
         super(EmailResetForm, self).__init__(*args, **kwargs)
 
+    def clean_email(self):
+        email = self.cleaned_data["email"]
+        try:
+            User.objects.get(email__iexact=email)
```

Comment:
That's a good idea, I extended Django's `UserManager` by adding this method.
https://github.com/MTG/freesound/pull/763/commits/50a73c5e3d6d0a049eec45f4a8f4a0122f59d1bf